server/daos/books.js

```diff
@@ -2,5 +2,26 @@ import db from '@database/models';
 
 export const insertBook = args => db.books.create(args);
 
-export const updateBook = async ({ id, name, genres, pages, publisherId }) =>
-  db.books.update({ name, genres, pages, publisherId }, { where: { id } });
+export const updateBook = async ({ id, name, genres, pages, publisherId }, { fetchUpdated }) => {
```

Comment:
let's create a function that does this, that all the "update" functions can call. There seems to be a considerable amount of duplication here

Reply:
okay